app/models/User.coffee

```diff
@@ -367,12 +367,17 @@ module.exports = class User extends CocoModel
     return 'enrolled' unless coursePrepaid.endDate
     return if coursePrepaid.endDate > new Date().toISOString() then 'enrolled' else 'expired'
 
-  prepaidType: ->
+  prepaidType: (includeCourseIDs) =>
     # TODO: remove once legacy prepaidIDs are migrated to objects
     return undefined unless @get('coursePrepaid') or @get('coursePrepaidID')
+    type = @get('coursePrepaid')?.type
+    if includeCourseIDs
+      courses = @get('coursePrepaid')?.includedCourseIDs
+      if type == 'course' and Array.isArray(courses)
+        return (courses.map (id) -> utils.courseAcronyms[id]).join('+')
```

Comment:
I am a little confused. Doesn't the `coursePrepaid` already contain the information of what courses are included?
Could we use a call to the prepaid `typeDescription` method? May need to create the model.

Reply:
@AndrewJakubowicz  oh, yes. this logic is a little ugly. But `coursePrepaid` is little different with `Prepaid`, it copies key-value from `Prepaid` but cannot access methods of `Prepaid` model. And since nick suggest me not create a new type for prepaid, so prepaid type is only 'course' and 'starter-license'. so the way to determine whether a 'course' prepaid is actual a 'customized' prepaid, we should check the `includedCourseIDs` key. this key is array when 'customized license', and 'null' when 'full license'. 

i think this logic can handle our needs. and creating a new model for user's `coursePreapid` may break a lot of exists logic.

Reply:
I have a hard time following everything going on here, but +1 on the point to avoid additional complexity with `Prepaid` and new models.

Reply:
Good call. You can probably think of a more informative comment, but something along these lines may be useful?

```
Use argument for customized course ID's
Be careful to match the returned string EXACTLY to avoid comparison issues
```